src/site/content/en/blog/origin-trials/index.md

```diff
@@ -65,12 +65,17 @@ Find out more: [What are third-party origin trials?](/third-party-origin-trials)
 1. When your token expires, you will get an email with a renewal link.
    To do so, you are again asked to submit feedback.
    
-{% Aside 'warning' %}
-Even if an API lands unchanged after a successful origin trial, there is a short period between the 
+{% Aside %}
+Usually if an API lands unchanged after a successful origin trial, there is a short period between the 
```

Comment:
Why remove the warning? This seems like a good use for it.

Reply:
Good spot — I think it should stay.

Reply:
Agreed. My initial thinking was it'd be just a "note" now, but in the general case OTs will *not* launch without interruption, so the "warning" still makes sense.